helmclient.go

```diff
@@ -815,23 +816,31 @@ func getTillerImage(pod *corev1.Pod) (string, error) {
 	return tillerImage, nil
 }
 
-func isTillerOutdated(pod *corev1.Pod) (bool, error) {
-	currentTillerImage, err := getTillerImage(pod)
+func validateTillerVersion(pod *corev1.Pod, desiredImage string) error {
```

Comment:
Yes changing the method signature back makes sense. As it gets called from newTunnel.